climbing_stairs/climbing_stairs.py

```diff
@@ -2,23 +2,35 @@
 
 import sys
 
+
```

Comment:
Great job implementing a cache to performance optimize this! I was able to run both tests in 0.002 seconds